backend/dal/Migrations/v01.06.03/Up/PostUp/01-ViewProperties.sql

```diff
@@ -0,0 +1,219 @@
+PRINT 'Updating View_Properties'
+GO
+-- This script provides a way to union both parcels and buildings into a single result of properties.
+-- Updating view to support many-to-many parcels and buildings.TEMPORARY
+CREATE OR ALTER VIEW dbo.[View_Properties] AS
+SELECT
+    p.[Id]
+    , [PropertyTypeId] = 0
+    , p.[ClassificationId]
+    , [Classification] = c.[Name]
+    , p.[AgencyId]
+    , [Agency] = ISNULL(pa.[Name], a.[Name])
+    , [AgencyCode] = ISNULL(pa.[Code], a.[Code])
+    , [SubAgency] = (
+        SELECT CASE WHEN pa.[Id] IS NOT NULL
+            THEN a.[Name]
+            ELSE NULL
+        END)
+    , [SubAgencyCode] = (
+        SELECT CASE WHEN pa.[Id] IS NOT NULL
+            THEN a.[Code]
+            ELSE NULL
+        END)
+    , p.[AddressId]
+    , [Address] = TRIM(ISNULL(adr.[Address1], '') + ' ' + ISNULL(adr.[Address2], ''))
+    , [AdministrativeArea] = adr.[AdministrativeArea]
+    , [Province] = ap.[Name]
+    , adr.[Postal]
+    , p.[ProjectNumbers]
```

Comment:
Is this the only change `ProjectNumbers`?

Reply:
yeah, just projectnumbers